packages/swapper/src/swappers/JupiterSwapper/swapperApi/getTradeQuote.ts

```diff
@@ -131,13 +154,59 @@ export const getTradeQuote = async (
         })
       : { instruction: undefined, destinationTokenAccount: undefined }
 
+  const buyAssetAddress =
+    buyAsset.assetId === solAssetId
+      ? fromAssetId(wrappedSolAssetId).assetReference
+      : fromAssetId(buyAsset.assetId).assetReference
+
+  const sellAssetAddress =
+    sellAsset.assetId === solAssetId
+      ? fromAssetId(wrappedSolAssetId).assetReference
+      : fromAssetId(sellAsset.assetId).assetReference
+
+  const [buyAssetReferralPubKey] = await PublicKey.findProgramAddressSync(
```

Comment:
```suggestion
  const [buyAssetReferralPubKey] = PublicKey.findProgramAddressSync(
```

Reply:
Done in [10c98b6](https://github.com/shapeshift/web/pull/8267/commits/10c98b6156314e71f8ec309e25bbd54fd475ec53)